Example/AppDelegate.m

```diff
@@ -0,0 +1,47 @@
+//
```

Comment:
Why are we adding an entire project here?

Reply:
Its a UI to test this solution.